typings.d.ts

```diff
@@ -92,6 +98,8 @@ export function addRowAt(rowIndex: number, clonePreviousRow?: boolean): (tr: Tra
 
 export function cloneRowAt(cloneRowIndex: number): (tr: Transaction) => Transaction;
 
+export function copyRow(insertNewRowIndex: number, rowToBeClonedIndex: number, options?: CopyRowOptions): (tr: Transaction) => Transaction;
```

Comment:
this is very a confusing API to have two functions `cloneRowAt` and `copyRow` in the library, lets keep only one

Reply:
Agree, but I don't have a better name and I don't agree about make `cloneRowAt` be a breaking change now.